README.md

```diff
@@ -26,33 +26,16 @@ button.
 
 ### Install from Source
 
-Before installation, you must have both [cmake][] and [Microsoft Visual C++][]
-installed.
-
-1.  Download the [EditorConfig C core][] and follow the instructions in the README
-    and INSTALL files to compile and install it with Microsoft Visual C++. Note that the
-    EditorConfig C core must be built with the `MT` option.
+Before installation, you must have both [cmake][] and [Microsoft Visual Studio][] installed.
 
+1.  Download the [EditorConfig C core][] and follow the instructions in the README and INSTALL files to compile and install it with Microsoft Visual C++. Note that the EditorConfig C core must be built with the `MT` option. To build the x64 version of the plugin, the x64 version of EditorConfig C core is needed.
 2.  Download the [EditorConfig plugin for Notepad++][] and extract it.
+3.  Open `cmake-gui`, select the extracted folder as source code directory, and create a directory to build the binaries. Then click the `Configure` button. When asked for a generator, select a generator without suffix to build the x86 version, or a generator with `Win64` suffix to build the x64 version.

```

Comment:
Do you wanna say the BUILD_TYPE should be set to `Release`?

Reply:
1. I think `Release` should be used to compile the dll for daily use. Release builds should be smaller and runs faster than Debug builds. However using the Debug one is also fine.
2. There are 2 places where we can change the option. The first place is when we "Configure" the CMake project, we can specify a set of available configuration types in the `CMAKE_CONFIGURATION_TYPES` option. The second place is when we opened the project using VS, there is a drop down menu on the toolbar so we can select the configuration from the set.

   `CMAKE_CONFIGURATION_TYPES` defaults to `Debug;Release;MinSizeRel;RelWithDebInfo` and VS defaults to the first one, which is `Debug`.
3. Maybe I should call it "configuration type" instead of "build type". Do I need to change it or just remove the suggestion to use `Release`?

Reply:
1. Please just mention the existence of `Release`. Otherwise, many people may forget.
2. Either place is OK.
3. Sure, as long as we use some name that is consistent with what is shown in the cmake GUI/VS, it is OK.